docs/intro/clients.rst

```diff
@@ -94,6 +95,16 @@ Configure the environment as needed for your preferred language.
 
     $ dotnet new console -o . -f net6.0
 
+  .. code-tab:: bash
+    :caption: Maven (Java)
+
+    $ touch Main.java
+
+  .. code-tab:: bash
+    :caption: Gradle (Java)
+
+    $ touch Main.java
```

Comment:
```suggestion
    :caption: Java

    $ touch Main.java
```

Reply:
The one nice thing about having both is that, if the tab labels match, the docs will automatically switch all of the tabbed sections at once. So, when I select the "Maven (Java)" tab in this section, it will automatically select that tab in the following tabs as well. I understand it doesn't make much sense otherwise, but do you think that's a worthwhile reason to keep both or would it just lead to confusion?